conf/application.conf

```diff
@@ -93,10 +93,12 @@ microservice {
     }
 
     features {
+      list-of-available-response-error-explanation-modes: "CDS, Exports"
```

Comment:
Is this config value used?

Reply:
It's not, but I found a similar solution in https://github.com/hmrc/app-config-qa/commit/b75be60f7165d4c2b1de101b217b2936d9769049#diff-88ec3c9b73885d4eacf1d4667e69651e
This is more of an info for someone changing the config, what values they can use.

Reply:
I think it can be removed, because journey we need to have different based on the environment, it's affect user point of view because he will see more choices or less. Errors are our internal stuff. We're not using this and the point of showing specific error is based on the feature flag.

I don't want to keep this, because if it's unused it can be difficult to find the reason of creation in few weeks or it will be difficult to decide to remove it because it can be somehow used. I remember last ticket from @SteveSugden where it was difficult to decide what part we can remove